mne/stats/regression.py

```diff
@@ -275,24 +271,30 @@ def solver(X, y):
     return evokeds
 
 
-def _prepare_rerp_data(raw, events, picks=None, decim=1):
+def _prepare_rerp_data(raw, events, picks=None, decim=1, sfreq=None):
     """Prepare events and data, primarily for `linear_regression_raw`. See
     there for an explanation of parameters and output."""
```

Comment:
Also, maybe it would make sense to update linear_regression_raw to be less reliant on MNE classes, and more on bare np arrays anyways.